invenio_administration/assets/semantic-ui/js/invenio_administration/details/DetailsComponent.js

```diff
@@ -0,0 +1,69 @@
+// This file is part of InvenioAdministration
+// Copyright (C) 2022 CERN.
+//
+// Invenio RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import PropTypes from "prop-types";
+import React, { Component, useContext } from "react";
+import Overridable from "react-overridable";
+import { Table } from "semantic-ui-react";
+import AdminDetailsContext from "./AdminDetailsContext";
+
+function DetailsComponent({ ...props }) {
+  const ctx = useContext(AdminDetailsContext);
+  return <Element data={ctx.data} {...props} />;
+}
+
+const Element = ({ ...props }) => {
+  return (
+    <Overridable id="DetailsComponent.table">
```

Comment:
This is the only component that is currently being rendered in the details page body. Therefore, it is the only one that is overridable.